index.js

```diff
@@ -147,6 +147,9 @@ alexa.response = function(session) {
     self.response.response.directives.push(audioPlayerDirective);
     return this;
   };
+  this.directive = function (directive) {
+    self.response.response.directives.push(directive);
```

Comment:
This should return `this` like every other method probably?

The methods above for `audioPlayer` should reuse this method.